tasmota/include/tasmota_template.h

```diff
@@ -181,6 +181,7 @@ enum UserSelectablePins {
   GPIO_OPTION_E,                       // Emulated module
   GPIO_SDM230_TX, GPIO_SDM230_RX,      // SDM230 Serial interface
   GPIO_ADC_MQ,                         // Analog MQ Sensor
+  GPIO_ADC_TDS,                         // Analog TDS Sensor
```

Comment:
New GPIOs must be added at the end of the list as stated in the comment lines 23-24

Reply:
Ok